functions/go/render-helm-chart/third_party/sigs.k8s.io/kustomize/api/builtins/HelmChartInflationGenerator.go

```diff
@@ -135,12 +133,28 @@ func (p *HelmChartInflationGeneratorPlugin) runHelmCommand(
 	return stdout.Bytes(), err
 }
 
-// createNewMergedValuesFile replaces/merges original values file with ValuesInline.
-func (p *HelmChartInflationGeneratorPlugin) createNewMergedValuesFile() (
-	path string, err error) {
+// createNewMergedValuesFiles replaces/merges original values file with ValuesInline.
+func (p *HelmChartInflationGeneratorPlugin) createNewMergedValuesFiles(path string) (
+	string, error) {
+	pValues, err := ioutil.ReadFile(path)
+	if err != nil {
+		if u, err := url.Parse(path); err == nil && (u.Scheme == "http" || u.Scheme == "https") {
+			var hc *http.Client
+			hc = &http.Client{}
+			resp, err := hc.Get(path)
```

Comment:
No need to create a client here. You can simply do `http.Get(path)`

Reply:
done